<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewFilesPage</class>
 <widget class="QWidget" name="ViewFilesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>fourohfour Files - View</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#ViewFilesPage {
    background-color: rgb(231, 236, 239);
}

QFrame#topHeaderFrame {
    background-color: white;
    border-bottom: 1px solid #E0E0E0;
}

QFrame#leftSidebarFrame {
    background-color: white;
    border-right: 1px solid #E0E0E0;
}

QPushButton#sharedFilesButton,
QPushButton#ownedFilesButton {
    background-color: transparent;
    border: none;
    text-align: left;
    padding: 10px 15px;
    color: #424242;
    font-size: 10pt;
    border-radius: 4px;
    font-weight: bold;
}

QPushButton#ownedFilesButton {
    background-color: #E3F2FD;
    color: #2196F3;
}

QPushButton#sharedFilesButton:hover,
QPushButton#ownedFilesButton:hover {
    background-color: #F5F5F5;
}

QListWidget#listWidget {
    background-color: rgb(255, 255, 255);
    border: 1px solid #E0E0E0;
    border-radius: 4px;
}
QListWidget::item {
    padding: 0px;
    height: 48px;
}
QListWidget::item:selected {
    background-color: #E3F2FD;
    color: #424242;
}


.header-item {
    background-color: #616161;
    color: white;
    font-weight: bold;
    font-size: 11pt;
}

.data-item {
    background-color: white;
    color: #424242;
    font-size: 10pt;
}

QPushButton#backButton {
    background-color: #757575;
    color: white;
    border: none;
    border-radius: 4px;
    min-width: 80px;
    min-height: 35px;
    font-size: 10pt;
    font-weight: bold;
}
QPushButton#backButton:hover {
    background-color: #616161;
}
QPushButton#backButton:pressed {
    background-color: #424242;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="topHeaderFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="topHeaderLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="appLogoLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#2196F3;&quot;&gt;fourohfour&lt;/span&gt;&lt;span style=&quot; font-size:18pt; color:#424242;&quot;&gt; Files&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="topHeaderSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1000</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="userAvatarLabel">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/images/user_avatar.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="contentHorizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="leftSidebarFrame">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="sidebarVerticalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="ownedFilesButton">
          <property name="text">
           <string>Owned by me</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sharedFilesButton">
          <property name="text">
           <string>Shared with me</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="sidebarBottomSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="fileListContainerFrame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="fileListVerticalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="fileListTitleLayout">
          <item>
           <widget class="QLabel" name="viewFilesLabel">
            <property name="font">
             <font>
              <pointsize>24</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #424242;</string>
            </property>
            <property name="text">
             <string>Owned by me</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="fileListTitleSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="fileListStackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="ownedFilesPage">
           <layout class="QVBoxLayout" name="ownedFilesLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="ownedFilesHeaderWidget">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #F5F5F5; font-weight: bold;</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayoutOwnedHeader">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>8</number>
               </property>
               <property name="topMargin">
                <number>4</number>
               </property>
               <property name="rightMargin">
                <number>8</number>
               </property>
               <property name="bottomMargin">
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="nameLabelOwned">
                 <property name="text">
                  <string>Name</string>
                 </property>
                 <property name="fixedWidth">
                  <number>320</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="spacerAfterNameLabelOwned">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="sizeLabelOwned">
                 <property name="text">
                  <string>Size</string>
                 </property>
                 <property name="fixedWidth">
                  <number>60</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="ownerLabelOwned">
                 <property name="text">
                  <string>Owner</string>
                 </property>
                 <property name="fixedWidth">
                  <number>200</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="headerSpacerOwned">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="actionsLabelOwned">
                 <property name="text">
                  <string>Actions</string>
                 </property>
                 <property name="fixedWidth">
                  <number>90</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="ownedFilesListWidget">
              <property name="styleSheet">
               <string notr="true">QListWidget {
    background-color: rgb(255, 255, 255);
    border: 1px solid #E0E0E0;
    border-radius: 4px;
}
QListWidget::item {
    padding: 0px;
    height: 48px;
}
QListWidget::item:selected {
    background-color: #E3F2FD;
    color: #424242;
}
</string>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="flow">
               <enum>QListView::TopToBottom</enum>
              </property>
              <property name="resizeMode">
               <enum>QListView::Adjust</enum>
              </property>
              <property name="layoutMode">
               <enum>QListView::SinglePass</enum>
              </property>
              <property name="sortingEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="sharedFilesPage">
           <layout class="QVBoxLayout" name="sharedFilesLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="sharedFilesHeaderWidget">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #F5F5F5; font-weight: bold;</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayoutSharedHeader">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>8</number>
               </property>
               <property name="topMargin">
                <number>4</number>
               </property>
               <property name="rightMargin">
                <number>8</number>
               </property>
               <property name="bottomMargin">
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="nameLabelShared">
                 <property name="text">
                  <string>Name</string>
                 </property>
                 <property name="fixedWidth">
                  <number>320</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="spacerAfterNameLabelShared">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="sizeLabelShared">
                 <property name="text">
                  <string>Size</string>
                 </property>
                 <property name="fixedWidth">
                  <number>60</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="ownerLabelShared">
                 <property name="text">
                  <string>Owner</string>
                 </property>
                 <property name="fixedWidth">
                  <number>200</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="headerSpacerShared">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="actionsLabelShared">
                 <property name="text">
                  <string>Actions</string>
                 </property>
                 <property name="fixedWidth">
                  <number>90</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="sharedFilesListWidget">
              <property name="styleSheet">
               <string notr="true">QListWidget {
    background-color: rgb(255, 255, 255);
    border: 1px solid #E0E0E0;
    border-radius: 4px;
}
QListWidget::item {
    padding: 0px;
    height: 48px;
}
QListWidget::item:selected {
    background-color: #E3F2FD;
    color: #424242;
}
</string>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="flow">
               <enum>QListView::TopToBottom</enum>
              </property>
              <property name="resizeMode">
               <enum>QListView::Adjust</enum>
              </property>
              <property name="layoutMode">
               <enum>QListView::SinglePass</enum>
              </property>
              <property name="sortingEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="listBottomSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
